resources/assets/js/app.js

```diff
@@ -20,6 +20,8 @@ window.axios.defaults.headers.common = {
     'X-Requested-With': 'XMLHttpRequest'
 };
 
+window.Vue.prototype.$http = window.axios;
```

Comment:
Please don't alias this in this PR. I'm happy to alias `$http` to `axios` but it doesn't seem fitting to do so in an unrelated PR.